app/views/api/v1/topics/_info.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.name       topic.name
+json.image      topic.image
```

Comment:
this can simplified with:

```
json.extract! topic, :name, :image
```
I personally prefer that way when I want to return the same attribute names that are on the object, its more scalable that having multiple lines (specially for models with multiple attributes)

Reply:
I also think it´s better and simpler. Initially, I did it this way becauses I was following the way done in other sections of the project.